requirements-xpu.txt

```diff
@@ -0,0 +1,11 @@
+# Common dependencies
+-r requirements-common.txt
+
+setuptools < 70.0.0 # IPEX's torch have some dependency. to be removed.
+
+torch @ https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_dev/xpu/torch-2.1.0.post1%2Bcxx11.abi-cp310-cp310-linux_x86_64.whl
```

Comment:
btw, torch 2.1 is a bit old now. Do you have a new version of these libraries?

Reply:
Not yet. As far as I know, ipex team is working on support torch 2.3 in next release.

Reply:
Thanks for letting us know! Please upgrade the dependency once the release is out :)